### PR TITLE
Recognize the application/CDFV2 content type

### PR DESCRIPTION
**Pull request by Magomogo**
CDFV2 is Composite document file V2. We can't tell what is inside using libmagic, unfortunately.

FYI @smoskalenko

### CODE REVIEW

**Comment by Magomogo** [comment]
As libmagic v can recognize the file, I'll try to make other versions to work in the same way. WIP.

**Comment by Magomogo** [comment]
Behaviour depend on the PHP version, not libmagic. Nothing to do, let's merge.

**Review by kevich — APPROVED** [review]

File: src/Barberry/ContentType.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ class ContentType
         'ots' => 'application/vnd.oasis.opendocument.spreadsheet-template',
         'ods' => 'application/vnd.oasis.opendocument.spreadsheet',
         'txt' => 'text/plain',
-        'xls' => 'application/vnd.ms-excel',
+        'xls' => array ('application/vnd.ms-excel', 'application/vnd.ms-office', 'application/CDFV2'),
```
Comment (kevich) [inline src/Barberry/ContentType.php:62]: Spacing? And maybe convert to short syntax everywhere?)

Comment (Magomogo) [reply src/Barberry/ContentType.php]: Improved. Ready for review.

**Pull request merged**